frontend/src/features/recommendation/UploaderFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box, Button } from '@material-ui/core';
+import { FilterSection } from 'src/components';
+
+interface Props {
+  value: string;
+  onChange: () => void;
+}
+
+function UploaderFilter(props: Props) {
+  return (
+    <FilterSection title="Uploader">
+      <Box
+        display="flex"
+        flexDirection="column"
+        style={{ textAlign: 'center' }}
+      >
+        <span style={{ padding: 4 }}>Uploader: {props.value}</span>
+        <Button
+          type="submit"
+          size="large"
+          variant="contained"
+          color="primary"
+          onClick={props.onChange}
+        >
+          <span>Reset</span>
+        </Button>
+      </Box>
+    </FilterSection>
```

Comment:
Instead of adding a new `<FilterSection>` that is a bit cumbersome in the existing layout, it could be a very good usecase for a deletable [`<Chip>`](https://v4.mui.com/components/chips/#chip-2).  

Then, it could be displayed above other filters, with minimal modification:
![image](https://user-images.githubusercontent.com/4726554/146407348-b1350e52-7665-4096-bfeb-59444bdaf9f2.png)



And the implementation is simple enough:
```tsx
    <Chip
      label={
        <Typography variant="body2">
          <strong>Uploader: </strong>
          {props.value}
        </Typography>
      }
      color="secondary"
      onDelete={props.onChange}
    />
```

What do you think @sandre35?